swatch-billable-usage/build.gradle

```diff
@@ -31,6 +31,11 @@ dependencies {
     implementation project(":swatch-model-billable-usage")
     annotationProcessor enforcedPlatform(libraries["quarkus-bom"])
     annotationProcessor "org.hibernate.orm:hibernate-jpamodelgen"
+    implementation libraries["mapstruct"]
+    annotationProcessor libraries["mapstruct-processor"]
+    testAnnotationProcessor libraries["mapstruct-processor"]
+    // if you are using mapstruct in test code
+    annotationProcessor libraries["lombok-mapstruct-binding"]
```

Comment:
I don't think we need the lombok-mapstruct-binding and `testAnnotationProcessor libraries["mapstruct-processor"]`.
I tried to remove them and the build worked fine (including tests).

Reply:
Interesting! The build is passing for me with it. But, I see we are not currently using it so, may as well remove it.

Reply:
Done